pkg/server/commands/list_objects.go

```diff
@@ -231,6 +254,16 @@ func (q *ListObjectsQuery) evaluate(
 			graph.WithMaxConcurrentReads(q.maxConcurrentReads),
 		)
 
+		if q.checkCache != nil {
+			cachedCheckResolver := graph.NewCachedCheckResolver(
```

Comment:
We're not using the `cachedCheckResolver` below when we call `Check`, so we're not actually using the cache. You should follow the same pattern you followed in Check here.

https://github.com/openfga/openfga/pull/891/files#diff-104a11712684e520f0affa95e6a053faeb713306a10f69ee99b9729cc9dd1996R413-R429